cluster-autoscaler/core/scale_up.go

```diff
@@ -430,7 +430,7 @@ func ScaleUp(context *context.AutoscalingContext, processors *ca_processors.Auto
 		if context.MaxNodesTotal > 0 && len(nodes)+newNodes+len(upcomingNodes) > context.MaxNodesTotal {
 			klog.V(1).Infof("Capping size to max cluster total size (%d)", context.MaxNodesTotal)
 			newNodes = context.MaxNodesTotal - len(nodes) - len(upcomingNodes)
-			context.LogRecorder.Eventf(apiv1.EventTypeWarning, "MaxNodesTotalReached", "Max total nodes in cluster reached: %v", newNodes)
+			context.LogRecorder.Eventf(apiv1.EventTypeWarning, "MaxNodesTotalReached", "Max total nodes in cluster reached")
```

Comment:
Would it make sense to report what's the total number of nodes instead how many new got added?

Reply:
Done in https://github.com/openshift/kubernetes-autoscaler/compare/bb52bd71348a22de465e219cc74dd18e19be180b..d65c26fb29ec3858c5213704d6d51804061fbc70